src/main/java/com/glodblock/github/inventory/FluidConvertingInventoryAdaptor.java

```diff
@@ -274,15 +276,16 @@ public boolean containsItems() {
         boolean anyValid = false;
         for (ForgeDirection dir : ForgeDirection.VALID_DIRECTIONS) {
             // Avoid sending stuff into itself me network
-            if (checkValidSide(this.posInterface.getOffSet(dir).getTileEntity(), dir)
-                    && !isItemConduit(this.posInterface.getOffSet(dir).getTileEntity())) {
-                final int result = checkItemFluids(this.getSideFluid(dir), this.getSideItem(dir), dir.getOpposite());
-                if (result == 1) {
-                    return true;
-                }
-                if (result != 2) {
-                    anyValid = true;
-                }
+            TileEntity te = this.posInterface.getOffSet(dir).getTileEntity();
+            if (te == null || !checkValidSide(te, dir)
+                    || (isConduit(te) && (isItemConduit(te) || !isFluidConduitConnected(te, dir))))
+                continue;
+            final int result = checkItemFluids(this.getSideFluid(dir), this.getSideItem(dir), dir.getOpposite());
```

Comment:
Generally, I woud prefer if we didnt use continue.

Reply:
What's wrong with `continue` now? It's used incredibly frequently in java

Reply:
no continue anymore